tools/src/common/mod.cpp

```diff
@@ -55,6 +57,9 @@ tMod::tMod(const std::string &szFileName)
 
 		// Data read successfully, fill sample info
 		tSample Sample;
+		for (uint8_t i = 0; i < SAMPLE_NAME_SIZE; ++i){ 
```

Comment:
we already have `i` variable in `for` loop above, it's best to not redeclare variables like that. Perhaps it's better to call it something like `c` for char since we're iterating on chars or `ubCharIndex` if you feel like it.